spec/router_spec.js

```diff
@@ -142,6 +143,68 @@ describe("Router", function() {
     });
   });
 
+  describe("navigate", function() {
+    class MyRouter extends Router {
+    }
+
+    class MyView extends View {
+      render() {
+        this.$el.html('');
+        return this;
+      }
+    }
+
+    beforeEach(function() {
+      this.router = new MyRouter({ $el: $('<div>') });
+    });
+
+    it("should call router.navigate right away if view does not define a beforeNavigate function", function() {
```

Comment:
The "if view does not define a beforeNavigate function" fragment describes a context and is perfect for folding into a `describe` block. 

```javascript
describe("navigate", function() {
  describe("with activeView, without beforeNavigate", function() {
    it("calls router.navigate right away", ...
  });
});

Reply:
Having written that comment ^, it's clear we're missing tests testing a "without active view" context.